actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -157,50 +163,76 @@ pub fn call<BS: Blockstore, RT: Runtime<BS>>(
             let dst_addr: EthAddress = dst.try_into()?;
             let dst_addr: Address = dst_addr.try_into()?;
 
-            let call_result = match kind {
-                CallKind::Call => system.send(
-                    &dst_addr,
-                    // readonly is sticky
-                    if system.readonly {
-                        Method::InvokeContractReadOnly
+            // Special casing for embryo/non-existent actors: we just do a SEND (method 0)
+            // which allows us to transfer funds (and create embryos)
+            let is_embryonic = {
```

Comment:
I am sure this can be rewritten in some funky way that has no ifs, because crabs don't like them...

Aside from that, should we allow non-existent id addrs or is that a bug?

Reply:
Added some logic to not consider a non-existent ID address embryonic, as this is not predictable in general.

Reply:
yeah non-existent id addrs are not embryos, embryos should only exist in the context of f4 addresses + abstract accounts, but this is a hack for now...